.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     'plugin:@typescript-eslint/recommended', // Plugin to use typescript with eslint
     'prettier', // Add prettier rules to eslint
     'plugin:prettier/recommended', // Plugin to use prettier rules with eslint
+    'plugin:react/jsx-runtime', // Add runtime rules for React 17
```

Comment:
This is related to https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

tl;dr we don't need to import react anymore but eslint is still complaining